bench/tiered/push_pull.c

```diff
@@ -254,11 +254,11 @@ remove_local_cached_files(const char *home)
 
         ++nmatches;
 
-        sscanf(dir_entry->d_name, "%*[^0-9]%d", &objnum);
+        testutil_assert(sscanf(dir_entry->d_name, "%*[^0-9]%d", &objnum) == 1);
         highest = WT_MAX(highest, objnum);
     }
 
-    closedir(dir);
+    testutil_check(closedir(dir));
```

Comment:
I feel like we have an opportunity here to enhance the `testutil_` macros. Specifically in the case where the API call returns a negative value (typically -1) and sets `errno`. For such API calls, the specific error code and therefore context will not be available.

The `closedir()` calls is not a good example of this as there is only one errno value (on Linux `EBADF`); but look at the man page  for `close()`.

Reply:
Thank you for the suggestion! I went with `testutil_assert_errno`